source/_docs/load-and-performance-testing.md

```diff
@@ -1,200 +1,75 @@
-
-<div class="alert alert-info" role="alert">
-<h3 class="info">Note</h3>
-<p>Testing with a session cookie to emulate the experience of a logged-in user is extremely important, as the contrast between an anonymous user and a logged-in user may be drastically different.</p></div>
-
-## Performance Goals
-
-Response times vary from site to site depending on the size of your modules stack, database queries, etc. Generally, anything under 1 second is considered excellent, but this is up to you.
-
-Emulating a logged in user's experience with `ab` is a key metric, as it provides the number of pages per second your site can generate on Pantheon. This number may determine whether or not you need to add additional application containers.
-
-## Testing Tools
-
-There are a number of other tools to consider when you are planning your load testing strategy. This can vary by the need for detail, nature of your site, or requirements for quality analysis.
-
-<table class="table">
-<tbody>
-		<tr>
-			<th>Testing Tool</th>
-			<th>Documentation</th>
-			<th>Acquisition</th>
-		</tr>
-		<tr>
-			<td>Apache AB</td>
-			<td><a href="http://httpd.apache.org/docs/2.2/programs/ab.html">Documentation</a></td>
-			<td><a href="http://httpd.apache.org/download.cgi">Download</a></td>
-		</tr>
-		<tr class="tr_class1">
-			<td>J-Meter</td>
-			<td><a href="http://jmeter.apache.org/usermanual/index.html">Documentation</a></td>
-			<td><a href="http://jmeter.apache.org/download_jmeter.cgi">Download</a></td>
-		</tr>
-		<tr>
-			<td>The Grinder</td>
-			<td><a href="http://grinder.sourceforge.net">Documentation</a></td>
-			<td><a href="http://grinder.sourceforge.net/download.html">Download</a></td>
-		</tr>
-		<tr>
-			<td>Blitz.io</td>
-			<td><a href="http://blitz.io/docs/">Documentation</a></td>
-			<td><a href="https://www.blitz.io/pricing#/subscriptions">Pricing</a></td>
-		</tr>
-	</tbody>
-</table>
+Load and performance tests are critical steps in going live procedures, as they help expose and identify potential performance killers. These tests provide insight for how a site will perform in the wild under peak traffic spikes.
+
+## Load vs Performance Testing
+Before you start, it's important to understand the difference between load and performance testing and know when to use each.
+### Performance Testing
+Performance testing is the process in which you measure an application's response time to proactively expose bottlenecks. These tests should be regularly executed as part of routine maintenance. Additionally, you should run these test before any load testing. If your application is not performing well, then you can be assured that the load test will not go well.  

```

Comment:
In general, I'd favor suggesting that clients: 

- "**refer to New Relic reports regularly** to identify improvements or degradation of performance
- "**perform performance test occasionally** to proactively exposed potential bottlenecks and to identify opportunities for optimization" and to 
- perform load tests in advance of anticipated major-traffic events, or prior to launching sites after major overhauls, remembering to provide enough time to fix any issues identified".